src/tasks/AndroidAppBuilder/Templates/monodroid.c

```diff
@@ -216,6 +225,22 @@ mono_droid_runtime_init (const char* executable, int managed_argc, char* managed
     appctx_values[0] = ANDROID_RUNTIME_IDENTIFIER;
     appctx_values[1] = bundle_path;
 
+    char* file_name = RUNTIMECONFIG_BIN_FILE;
+    int str_len = strlen (bundle_path) + strlen (file_name) + 1;
+    char* file_path = (char *) malloc (sizeof (char) * (str_len + 1));
+    int res1 = snprintf (file_path, str_len, "%s/%s", bundle_path, file_name);
```

Comment:
I'm not particularly fond of `res1` as the name here.

Reply:
How about `res_file_path`?

Reply:
Changed to `num_char`.